Cargo.toml

```diff
@@ -8,10 +8,11 @@ license = "MIT"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-dogstatsd = { version = "0.8", default-features = false }
+dogstatsd = { version = "0.8.1", default-features = false }
```

Comment:
I don't really like this, but then again they shouldn't have added something in a patch...

Reply:
Yes, there are the changes to `time` and `async_time` functions that returns the block result if there's an error as `Result<BlockOutT, (BlockOutT, Err)>`